charts/scalar-manager/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: scalar-manager
+description: Scalar Manager
+type: application
+version: 1.0.0
+appVersion: 1.0.0
```

Comment:
The Docker image is versioned from 1.0.0 and the Chart's version is aligned to the Docker image.